ui/app/AppLayouts/Chat/controls/community/HoldingTypes.qml

```diff
@@ -6,6 +6,6 @@ QtObject {
     }
 
     enum Mode {
-        Add, Update
+        Add, Update, UpdateOrRemove
```

Comment:
Is it something different `Update` or `UpdateOrRemove`? afaik they always go together in the designs
Could we keep just only 2 states/modes?

Reply:
Yes, when we reach given token via "+" that is already added, then it's Update (no remove button). When we reach added token via clicking on an item on the list, then it's "Update or remove" (+ remove button).

Reply:
Ok!! I didn't get this difference! Thanks!!